repo/fsrepo/migrations/fetcher.go

```diff
@@ -0,0 +1,83 @@
+package migrations
+
+import (
+	"context"
+	"io"
+	"os"
+)
+
+const (
+	// Current dirstibution to fetch migrations from
+	CurrentIpfsDist = "/ipfs/Qme8pJhBidEUXRdpcWLGR2fkG5kdwVnaMh3kabjfP8zz7Y"
+	// Distribution IPNS path.  Default for fetchers.
+	IpnsIpfsDist = "/ipns/dist.ipfs.io"
```

Comment:
Calling this IPNS when it's really DNSLink is pretty unfortunate, especially since things are already confusing given the `/ipns` prefix. How about referencing DNSLink or just using "Latest" like?

```suggestion
	// Latest distribution path.  Default for fetchers.
	LatestIpfsDist = "/ipns/dist.ipfs.io"
```'

Reply:
Done.